packages/components/src/components/header/header.component.ts

```diff
@@ -59,46 +60,60 @@ export default class SynHeader extends SynergyElement {
   private mutationObserver: MutationObserver;
 
   /**
-   * The headers label. If you need to display HTML, use the `default` slot instead.
+   * The headers label. If you need to display HTML, use the `label` slot instead.
    */
   @property() label = '';
 
   /**
-   * Adds a button to toggle the burger menu's visibility.
-   * The button is added automatically, if the component finds a syn-side-nav in non-rail mode.
+   * Defines the current visibility and icon of the burger-menu icon.
+   * The menu button is added automatically if the component finds a syn-side-nav in non-rail mode.
+   * The following values can be used:
+   * - hidden: The burger menu is not visible
+   * - opened: The burger menu is visible and shows the close icon
+   * - closed: The burger menu is visible and shows the open icon
    */
-  @property({ attribute: 'show-burger-menu', reflect: true, type: Boolean }) showBurgerMenu = false;
-
-  /**
-   * Determines whether or not the burger menu is currently visible.
-   */
-  @property({ attribute: 'burger-menu-visible', reflect: true, type: Boolean }) burgerMenuVisible = false;
+  @property({ attribute: 'burger-menu', reflect: true }) burgerMenu: 'hidden' | 'opened' | 'closed' = 'hidden';
```

Comment:
I think it should be "open" to fit to others components like "syn-detail".

Reply:
Fixed